destination-detail.php

```diff
@@ -0,0 +1,103 @@
+<?php require 'connection.php'; ?>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Destination Name</title>
+    <link rel="stylesheet" href="bootstrap.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <?php include 'navbar.php'; ?>
+
+            <!-- carousel -->
+            <div class="col-12 col-md-10 col-lg-8 offset-md-1 offset-lg-2 mt-3">
+                <div class="row">
+                    <div id="carouselExampleControls" class="carousel slide" data-bs-ride="carousel">
+                        <?php
+                        $desID = $_GET["desID"];
```

Comment:
You must validate the value before assigning it to the variable.

Reply:
done